solr/core/src/java/org/apache/solr/response/CSVResponseWriter.java

```diff
@@ -318,29 +318,27 @@ public void close() throws IOException {
   }
 
   //NOTE: a document cannot currently contain another document
-  @SuppressWarnings({"rawtypes"})
-  List tmpList;
+  List<Object> tmpList;
```

Comment:
unrelated to the changes in this pull request: could this be private and/or have a comment w.r.t. why it's not just a local variable in `writeSolrDocument` would be helpful